### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by BoKanchana**
add friend name

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,7 @@
 # EiEi
+
+Gif
```
Comment (jirapaparS) [inline README.md]: **Gift**

File: README.md [hunk 2]
```diff
@@ -1 +1,7 @@
...
+Po
+Pan
+B
+SupaDeerDar
```
Comment (DarkFairySM) [inline README.md]: Wrong word. this correct is Deer

File: README.md [hunk 3]
```diff
@@ -1 +1,7 @@
 # EiEi
+
+Gif
+Po
```
Comment (porichigo15) [inline README.md]: must be Por

File: README.md [hunk 4]
```diff
@@ -1 +1,7 @@
...
+Pan
```
Comment (Pang021) [inline README.md]: Pang

**Pull request merged**